io.github.srb2.rphys.yml

```diff
@@ -0,0 +1,103 @@
+app-id: com.github.rphys
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: srb2
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
```

Comment:
The game in GNOME wayland looks like this:

- No window decoration to drag
- No fullscreen

<details>
<summary>image</summary>

![image](https://github.com/flathub/flathub/assets/62639087/a00b3a04-cc7a-4667-91ab-c55a6982c602)
</details>

Is wayland actually supported? If not just use `--socket=x11`

Reply:
Wayland is supported. I'm not sure why the game looks like that on your system but I haven't touched SDL2 or rendering in my mod. The same issue should happen on the vanilla game's flatpak https://flathub.org/apps/org.srb2.SRB2